exercises/bowling/canonical-data.json

```diff
@@ -19,255 +19,773 @@
     "In all error cases you should expect an error as is idiomatic for your language",
     "whether that be via exceptions, optional values, or otherwise."
   ],
-  "cases": [{
-    "uuid": "656ae006-25c2-438c-a549-f338e7ec7441",
-    "description": "should be able to score a game with all zeros",
-    "property": "score",
-    "input": {
-      "previousRolls": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+  "cases": [
+    {
+      "uuid": "656ae006-25c2-438c-a549-f338e7ec7441",
+      "description": "should be able to score a game with all zeros",
+      "property": "score",
+      "input": {
+        "previousRolls": [
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ]
+      },
```

Comment:
This is awful. Can we configure it to not do this instead of ignoring the file altogether?

Reply:
I don't know how, other than adding a `.prettierrc` to this directory with a longer width.